package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fcl-web3auth-plugin",
-  "version": "0.2.2",
+  "version": "0.3.0",
```

Comment:
I'd make sure to reflect in the version that it's an alpha right now, here's an SO thread on that:

https://stackoverflow.com/questions/39206082/npm-version-to-add-alpha-postfix

fcl seems to follow that way of naming alpha versions as well: https://www.npmjs.com/package/@onflow/fcl?activeTab=versions